sdk/apis/tenancy/v1alpha1/types_workspace.go

```diff
@@ -174,6 +174,50 @@ type WorkspaceSpec struct {
 	//
 	// +kubebuilder:format:uri
 	URL string `json:"URL,omitempty"`
+
+	// Mount is a reference to a an object implementing a mounting feature. It is used to orchestrate
+	// where the traffic, intended for the workspace, is sent.
+	// If specified, logicalcluster will not be created and the workspace will be mounted
+	// using reference mount object.
+	//
+	// +optional
+	Mount *Mount `json:"mount,omitempty"`
+}
+
+func (in *WorkspaceSpec) IsMounted() bool {
+	return in.Mount != nil
+}
```

Comment:
This method is only used ~three times while `.Mount != nil` or equivalent is used more often.
Since `.Mount` cannot not be exported I'd remove this method to force using the same pattern everywhere this check happens.

Reply:
changed